src/aarch64/logic-aarch64.cc

```diff
@@ -36,33 +36,33 @@ namespace aarch64 {
 using vixl::internal::SimFloat16;
 
 template <typename T>
-bool IsFloat64() {
```

Comment:
Are the constexprs really required for compilation?

Reply:
The constexprs arn't required for compilation

If I remember correctly, I did those changes to tackle narrowing conversion warnings.
MSVC would warn for conversion in paths that wouldn't be taken for the type it was complaining about, However, MSVC is smart enough not to do this when constexprs are used.

Reply:
OK, sounds fine if it removes warnings using some static inference. I thought it might be an optimisation.